lambdaclass/mirra_backend
# [GH-776] Expose leaderboard endpoint

> [!Warning]
> Merge this pr alongside [This client pr](https://github.com/lambdaclass/champions_of_mirra/pull/1944)

## Motivation

We needed an endpoint to list our current users sorted by the total amount of prestige of the user units prestige

Closes #776

## Summary of changes

- added `/curse/users/leaderboard` endpoint to ask for the current leaderboard

## How to test it?

run `curl -X GET http://localhost:4001/curse/users/leaderboard` having the backend running and you should see a list of users with the total prestige that they have

## Checklist
